go.mod

```diff
@@ -54,7 +54,8 @@ require (
 	github.com/polyfloyd/go-errorlint v0.0.0-20201127212506-19bd8db6546f
 	github.com/ryancurrah/gomodguard v1.2.0
 	github.com/ryanrolds/sqlclosecheck v0.3.0
-	github.com/securego/gosec/v2 v2.6.1
+	github.com/sanposhiho/wastedassign v0.1.2
+	github.com/securego/gosec/v2 v2.5.0
```

Comment:
hmm, please revert the changes for `gosec`

Reply:
Oops, sorry.